docs/source/api/spatial-functions.rst

```diff
@@ -416,6 +416,60 @@ st_dump
     +-------------+
 
 
+st_envelope
+***********
+
+.. function:: st_envelope(col)
+
+    Returns the minimum bounding box of the input geometry, as a geometry.
```

Comment:
could we add here a comment that the returned geometry is a rectangle.
not to be confused by convex hull.

Reply:
@tdikland